modules/core/02-client/client/cli/query.go

```diff
@@ -89,6 +89,39 @@ func GetCmdQueryClientState() *cobra.Command {
 	return cmd
 }
 
+// GetCmdQueryClientStatus defines the command to query the status of a client with a given id
+func GetCmdQueryClientStatus() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "status [client-id]",
+		Short:   "Query client status",
+		Long:    "Query client activity status",
```

Comment:
Maybe add `Any client without an 'Active' status is considered inactive.` to the Long?

Reply:
Done